genesyscloud/resource_genesyscloud_tf_export.go

```diff
@@ -0,0 +1,594 @@
+package genesyscloud
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sync"
+
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const defaultTfJSONFile = "genesyscloud.tf.json"
```

Comment:
Minor style thing: this would usually be written as a const block
`const (
    defaultTFJSONFile = "genesyscloud.tf.json"
    defaultTFStateFile  = "terraform.tfstate"
)`